app.js

```diff
@@ -15,31 +15,34 @@ const profileRoutes = require("./API/profiles/routes");
 
 // Path
 const path = require("path");
-
+const bodyParser = require("body-parser");
+const eventRoutes = require("./API/Event/routes");
 const app = express();
 
 // Middleware
 app.use(cors());
 app.use(express.json());
 
+//Routes
+app.use("/events", eventRoutes);
+app.use(userRoutes);
+app.use("/profiles", profileRoutes);
+app.use("/media", express.static(path.join(__dirname, "media")));
+
 // Passport Setup
```

Comment:
Passport middleware must be called before the routes, so the routes could actually use it.